sdk/utils.ts

```diff
@@ -182,7 +182,7 @@ export const fetchJwtToken = async (claims: {}, expiresAt?: Date): Promise<{toke
     return response.data
   } catch(err) {
     if((err.message as string).includes("ENOTFOUND forta-jwt-provider")) {
-      throw Error("Could not resolve host 'forta-jwt-provider'. This url host can only be resolved inside of a running scan node") 
+      console.warn("Could not resolve host 'forta-jwt-provider'. This url host can only be resolved inside of a running scan node") 
     }
     throw err
```

Comment:
we're still potentially throwing an error here. i think we need to add code like this at the top of this method for local development:
```
if (!isProduction) return {token: "MOCK_JWT_TOKEN"}
...
```
then in production we can leave the error as-is

Reply:
Updated to only attempt to fetch a JWT when bot is running in production mode